packages/bpk-component-infinite-scroll/src/BpkInfiniteScroll.js

```diff
@@ -0,0 +1,172 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import 'intersection-observer';
+
+import { BpkSpinner, SPINNER_TYPES } from 'bpk-component-spinner';
+
+import STYLES from './BpkInfiniteScroll.scss';
+
+const infinitize = ComponentToExtend => {
```

Comment:
rename `infinitize` to `withInfiniteScroll`

Reply:
done